docs/core/sdk.md

```diff
@@ -22,7 +22,7 @@ It contains the following components:
 
 1. The .NET Core Command Line Tools that are used to build applications
 2. .NET Core (libraries and runtime) that allow applications to both be built and run
-3. The `dotnet` driver for running the CLI commands as well as running applications
+3. The `dotnet` driver for running the [CLI commands](tools/index.md) as well as running applications
 
 
 ## Acquiring the .NET Core SDK
```

Comment:
Maybe this section could be replaced with a link to the [tools page](https://docs.microsoft.com/en-us/dotnet/articles/core/tools/) -to shorten it- like:

Read more about Command Line Interface (CLI) tools, the installation process and how to use it in the [.NET Core Command Line Tools](tools/index.md) topic.